### PR TITLE
Updates the mobileapp setup so it works on iOS

### PR DESCRIPTION
**Pull request by madsnedergaard**
### Description

- Rewrites README to be more detailed
- Updates packages to work on newer iOS versions
- Removes Sentry - it didn't really work anyway and I couldn't get it building
- Fixes issue with gap at the bottom of the screen
- Changes the back-arrow to go back to map instead of the ranking-list

There's also some changes to the Android setup, but I'll follow up on that afterwards :)

### CODE REVIEW

File: web/views/pages/index.ejs [hunk 1]
```diff
@@ -68,6 +68,7 @@
         </script>
 
         <!-- Add crash reporting early on so we catch all errors -->
+        <% if (typeof isCordova === 'undefined' || !isCordova) { %>
```
[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/views/pages/index.ejs:71]: It should run on everything **except** the mobile app - so if `isCordova` is not set or `false`, we include Sentry. Isn't this logic doing that? 😅

[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/views/pages/index.ejs:71]: Oh wait, now I get it! Yes, you're right :D

File: web/src/layout/leftpanel/countrypanel.js [hunk 2]
```diff
@@ -196,7 +196,7 @@ const CountryPanel = ({
   const data = useCurrentZoneData() || {};
 
   const parentPage = {
-    pathname: isMobile ? '/ranking' : '/map',
```
Comment (Kongkille) [inline web/src/layout/leftpanel/countrypanel.js:199]: Is this separate from getting it to work? I know there's an issue where it keeps going back to ranking on the mobile version

Comment (Kongkille) [reply web/src/layout/leftpanel/countrypanel.js:199]: Great!

**Review by Kongkille — APPROVED** [review]

**Pull request merged**